app/src/main/java/ch/epfl/culturequest/storage/FireStorage.java

```diff
@@ -106,6 +107,19 @@ public static CompletableFuture<Profile> uploadNewProfilePictureToStorage(Profil
         return future;
     }
 
+    public static CompletableFuture<AtomicBoolean> deleteImage(String imageUrl) {
+        CompletableFuture<AtomicBoolean> future = new CompletableFuture<>();
+        StorageReference imageRef = storage.getReferenceFromUrl(imageUrl );
+        imageRef.delete().addOnCompleteListener(task -> {
+            if (task.isSuccessful()) {
+                future.complete(new AtomicBoolean(true));
+            } else {
+                future.complete(new AtomicBoolean(false));
+            }
+        });
+
+        return future;    }
+
```

Comment:
I believe that a null check on `imageUrl` could be necessary here.
I explain why in another comment for `ScanFragment.java`.